Firebase/Core/Private/FIRVersion.m

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Current convention is Private directory is only for headers. FIRVersion.m should be in Firebase/Core/

Reply:
Ah, got it. Done.